lib/processCss.js

```diff
@@ -42,12 +42,12 @@ var parserPlugin = postcss.plugin("css-loader-parser", function(options) {
 		}
 
 		if(options.import) {
-			css.walkAtRules(/import/i, function(rule) {
+			css.walkAtRules(/^import$/i, function(rule) {
 				var values = Tokenizer.parseValues(rule.params);
 				var url = values.nodes[0].nodes[0];
-				if(url.type === "url") {
+				if(url && url.type === "url") {
```

Comment:
This would be a new kind of test which tests for throwing errors. We do not do this currently in CSS loader it seems... at least not in the `importTest.js` file.